helm/nifi-cluster/Chart.yaml

```diff
@@ -30,7 +30,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.0.0
+version: 1.1.0
```

Comment:
I think this was missed in the release process. @juldrixx : Do you know how/where versions are updated when a release occurs? 

I don't see that releases are created via a circleCI job

Reply:
They are created manually. And it should be 1.1.1 not 1.1.0

Reply:
Apologies, I was versioning the helm chart separately to the overall nifikop release 😄 

I've pushed another commit to change this over to 1.1.1